src/applications/appeals/shared/components/NeedsToVerify.jsx

```diff
@@ -34,7 +34,7 @@ const NeedsToVerify = ({ pathname }) => {
           href={`/verify?next=${pathname}`}
           className="verify-link vads-c-action-link--green"
         >
-          Verify your identity
+          Verify your identity to start your request
```

Comment:
We have:
- "Request a Board Appeal"
- "Request a Higher-Level Review"
- "File a Supplemental Claim"

I don't know if "request" works for Supplemental Claim? @eileen-coforma what do you think?

Reply:
What's the context? On va.gov most of the references towards Supplemental Claims use "File a Supplemental Claim". It's safe to stick with that.

Reply:
![](https://user-images.githubusercontent.com/37054305/256942783-64c437fb-83f6-424d-b0c2-60b3b8af933c.png)

Reply:
For Supplemental Claims, do we want to update that action link to say `Verify your identity to file your claim`?

Reply:
I think for this, it's OK to say `Verify your identity to start your request` Thanks for the screenshot and extra context.